app/views/events/_form.html.erb

```diff
@@ -33,6 +33,8 @@
     <div class="col-lg-10">
       <%= f.text_field :max_participants, :class => 'form-control' %>
       <%=f.error_span(:max_participants) %>
+      <%= f.label :participants_are_unlimited %>
```

Comment:
I see your point. None of the solutions sounds nice, so we can keep this as well, I guess